### DISCUSSION TITLE
Export of Entire Query

### DISCUSSION THREAD

**Discussion by Marcello1099** [Budibase/budibase#11864] [Help] [answered]
Hello,

I am trying to make an export for an entire query. This can be either through email or through a direct download as well. As of now, I have made an automation to get the data from the query and then email it in plaintext. However, this list in the long term can become huge (> thousands). 

Therefore, is there any way I can either turn this email data to an attachment or can I download a query result directly? What would be my options to achieve this?

Thanks in advance!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Marcello1099,

I don't believe this is something which can be done currently. However you could possibly look into creating a custom automation step that would convert this to some kind of file and then use that custom automation step to output it in some other way e.g. to a bucket which can then be referenced to in the email step? [somewhere](https://docs.budibase.com/docs/custom-automations).

You can also create a feature request [here](https://github.com/Budibase/budibase/issues/new/choose) supply as much information as possible for it to get added to the core product.
